src/pages/workspaces/workspace/Data.js

```diff
@@ -322,25 +335,45 @@ const WorkspaceData = _.flow(
     )
   }
 
-  const loadMetadata = withErrorReporting('Error loading workspace entity data', async () => {
-    const [entityMetadata, { resources: snapshotMetadata }] = await Promise.all([
-      Ajax(signal).Workspaces.workspace(namespace, name).entityMetadata(),
-      Ajax(signal).Workspaces.workspace(namespace, name).listSnapshot(1000, 0)
-    ])
-
-    const snapshotEntities = await Promise.all(_.map(({ name: snapshotName }) => {
-      return Ajax(signal).Workspaces.workspace(namespace, name).snapshotEntityMetadata(namespace, snapshotName)
-    }, snapshotMetadata))
+  const loadEntityMetadata = async () => {
+    try {
+      setEntityMetadataError(false)
+      const entityMetadata = await Ajax(signal).Workspaces.workspace(namespace, name).entityMetadata()
+
+      setSelectedDataType(getSelectionType() === 'entities' && !entityMetadata[selectedDataType] ? undefined : selectedDataType)
+      setEntityMetadata(entityMetadata)
+    } catch (error) {
+      reportError('Error loading workspace entity data', error)
```

Comment:
this is user-facing language. Should it change to something like  "Error loading workspace tables" or "Error loading data tables" to be consistent with how the rest of the UI identifies it? "entity data" may not be clear to users.

Reply:
This is the same text as the previous error, so I'm gonna leave it as-is for now. I think there's a larger conversation about how we keep terminology in sync potentially to be had, so I'm not gonna poke it now.